new/LIT6350MEMiracle.xml

```diff
@@ -0,0 +1,90 @@
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2021-01-20">Created entity</change>
+            <change who="MV" when="2021-01-21">Added bibliography, fixed title</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Turaiev1905Eustathii"/>
+                        <citedRange unit="page"><!-- insert page range from Turaiev's Monumenta Aethiopiae Hagiologica: Vita et Miracula Eustathii, pp. 134-177 --> (1st miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+                <listBibl type="translation">
+                    <bibl>
+                        <ptr target="bm:Turaev1906Ewostatewos"/>
+                        <citedRange unit="page">71-72 (1st miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6350MEMiracle" passive="LIT4315Miracles"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BGV009"/>.</note>
+                    <ab>
+                        ፩ ተአምሪሁ፡ ለቅዱስ፡ አቡነ፡ ኤዎስጣቴዎስ፡ ዘይትብነበብ፡ አመ፡ ፲ወ፭ ለወርኃ፡ ታኅሣሥ።  <gap reason="ellipsis" resp="MV"/>
```

Comment:
፩ ተአምሪሁ፡ ለቅዱስ፡ አቡነ፡ ኤዎስጣቴዎስ፡ ዘይትብነበብ፡ አመ፡ ፲ወ፭ ለወርኃ፡ ታኅሣሥ። I wonder whether this part (as well as all the corresponding parts in the rest of the ms.) shall be encoded as "title expanded". What do you both think? For our catalogue I think it would be nice.